src/tests/test_cases.py

```diff
@@ -15,7 +15,7 @@ def filled_set():
 
 
 def test_contains(filled_set):
-    filled_set.add(5)
+    assert 4 == filled_set.add(5)
```

Comment:
We do not need to check the return value for `add` method there.
These are unit tests. Therefore this test case must only validate if `contains` function works properly.
We can even remove that line.